templates/quickstart-aurora-postgres-for-atlassian-services.yaml

```diff
@@ -583,7 +583,7 @@ Resources:
           - CreateSecurityGroup
           - [!Ref RDSSecurityGroup]
           - [!Ref CustomDBSecurityGroup]
-    DeletionPolicy: Retain
+    DeletionPolicy: Delete
```

Comment:
I don't think this is the right fix. In https://github.com/atlassian/quickstart-atlassian-services/commit/ebd2820214350ed6bb34df8cc0f7fadbb3ffa5e2 we have added the `Retain` value because we were not explicit with the `DeletionPolicy`. Before the `Retain` value it had the default value which is `Snapshot` based on this documentation:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-deletionpolicy.html
I think to revert back to the state before @tarka PR, we will need t change the `DeletionPolicy` to `Snapshot` and that should also make the cleanup easier. The cleaner lambda function contains a function to clean up stale Aurora snapshots.

We still need to make sure that instances created with the `Retain` policy can migrate to the new template version for backward compatability.

Reply:
@nanux  Are we not using `Delete` because we want to give customers to have a back up options even after clean up?